docs/src/main/tut/pull_request.md

```diff
@@ -85,12 +84,12 @@ If you want to create a pull request, we have two ways to create a pull request.
 
 On the one hand, we pass as parameters to create a new pull request:
```

Comment:
^^ There are two ways to create a pull request.

One the one hand, we can pass the following parameters: